proposals/NNNN-unlock-existential-types-for-all-protocols.md

```diff
@@ -76,11 +65,10 @@ Evidently, the current limitations on existential types prohibit the creation of
 
 ### Existential Type Synthesis for Library Authors
 
-Libraries often need to provide custom existential types for commonly-used protocols, when their generation is unavailable. However, that  introduces hard-to-understand and difficult-to-maintain boilerplate code into the codebase, not to mention that it imposes resilience limitations from the increase API and ABI surfaces, as well. As a result, library authors have to spend valuable time on creating, documenting and maintaining such code, which can easily be generated by the compiler. For instance, [SwiftUI](https://developer.apple.com/documentation/swiftui) relies heavily on the ubiquitous [`View`](https://developer.apple.com/documentation/swiftui/view) protocol. This `View` protocol has a public interface that approximately looks like this:
+For convenience, libraries are often bound to vend custom type-erased wrappers for commonly-used protocols when their existential counterpart is not supported. This comes at the expense of code size, resilience, and valuable time required to design, document, and maintain such code, more so if the target protocol is not owned by the library. For instance, [SwiftUI](https://developer.apple.com/documentation/swiftui) heavily relies on the ubiquitous [`View`](https://developer.apple.com/documentation/swiftui/view) protocol with a public interface approximating the following:
```

Comment:
`more so if the target protocol is not owned by the library`

I'm not sure if "owned" is the right word. Do you think that "is not declared in" would work better?